com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py

```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
+    """Raised when the SWS view element cannot be located."""
```

Comment:
What is a 'SWS view element' though?

As a user of the api I just care about a *view* and does it exist. SWS is just the mechanism used it bring the info back, and I don't know what a 'view element' is.